addons/web/static/src/js/core/dom.js

```diff
@@ -87,6 +87,18 @@ return {
             $textarea.css({height: Math.max(height + heightOffset, minHeight)});
         }
 
+        function preventVerticalResize() {
+            // We already compute the correct height:
+            // we don't want the user to resize it vertically.
+            // On Chrome this needs to be called after the DOM is ready.
+            var style = window.getComputedStyle($textarea[0], null);
+            if (style.resize === 'vertical') {
+                $textarea[0].style.resize = 'none';
+            } else if (style.resize === 'both') {
+                $textarea[0].style.resize = 'horizontal';
+            }
+        }
```

Comment:
i think i'd move this code at the end of resize()

Reply:
As we discussed, we don't need this code to trigger at every focus or change, just the first time.